api/v1alpha3/azurecluster_validation_test.go

```diff
@@ -485,6 +485,90 @@ func TestInternalLBIPAddressInvalid(t *testing.T) {
 	g.Expect(err.BadValue).To(BeEquivalentTo(internalLBIPAddress))
 }
 
+func TestIngressRuleValidPriority(t *testing.T) {
```

Comment:
just a style comment, I think these 3 new could be changed to a table test

Reply:
fixed this in an amended commit.